src/controllers/users.js

```diff
@@ -39,8 +38,8 @@ export default class usersController {
 
   static async verifyAcccount(req, res) {
     try {
-      const { token: userToken } = req.params;
-      const userInfo = jwt.decode(userToken, process.env.SECRETKEY);
+      const { token } = req.params;
```

Comment:
You should retrieve the token in headers, not in params

Reply:
@kagabof  here because we wanted to verify the token and be able to redirect to the fronted so we decided to pass it through the params maybe @EugeneMunya  can provide you more details on this.

Reply:
@Niyonsengaeric is right the email comes with token we have to sent back as routes of that endpoints suggest.

Reply:
But the best way to do that is to pass the token in headers. Please pass it in the headers for security issues.